### ISSUE TITLE
Forest can't render chart with smart field

### ISSUE THREAD

**Issue by glevha** [ForestAdmin/forest-rails#190]
Hello,

I have an issue rendering a very simple chart. I have an `orders` table with a `amount_cents` column (the total amount of the order in cents). I computed a full_amount smart fieldas following: 

```ruby
class Forest::Order
  include ForestLiana::Collection

  collection :orders

  field :full_amount, type: 'Number' do
    object.amount_cents.fdiv(100)
  end
end
```

It works well in my orders data in forest. 
However, when I try to use this smart field to display data in a chart, it doesn't work: `Forest cannot render this chart.`

I can't see what I am doing wrong, any idea ?

Thanks a lot for your great job !

**Comment by arnaudbesnier** [comment]
Hi @glevha,

It is definitely a UI issue in the chart creation panel.
You should not be able to use a Smart Field for charts creations.

If you want to create "complex" charts you can use Smart Charts (see the doc https://doc.forestadmin.com/developer-guide/rails.html#creating-a-smart-chart).

We'll fix this soon, I'll let you know.
Thanks for the feedback.

🌲🌲🌲

**Comment by glevha** [comment]
Hi @arnaudbesnier,

Ok thanks, I used Smart Charts to do it.

Gauthier

**Comment by arnaudbesnier** [comment]
Hi @glevha,
I am not sure to understand your last comment.
The issue you had appeared by creating a chart in the UI or did you do it using a Smart Chart with some specific code?

The reason I ask, is that I can't reproduce your issue with a Chart created in the Forest UI.

**Comment by arnaudbesnier** [comment]
@glevha can you help me to debug this?

**Comment by glevha** [comment]
Hi @arnaudbesnier, 

Sorry for the late answer. The issue I had appeared by creating a chart in the UI. To make it work, I finally created a Smart Chart.

When I create a simple chart, step 3 "Choose a data source", field "select a value", I can pick a smart field.

See attached picture : my "full amount" field is actually a smart field I created.

<img width="632" alt="capture d ecran 2017-09-18 a 09 58 11" src="https://user-images.githubusercontent.com/23697922/30533108-efe9558a-9c57-11e7-81f7-925b799c7a85.png">
